CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -242,7 +242,9 @@ public let DECL_NODES: [Node] = [
     base: .decl,
     nameForDiagnostics: "associatedtype declaration",
     documentation: """
-      An associated type declaration like the following.
+      An `associatedtype` declaration
+
+      For exmaple, an associatedtype declaration can look like the following.
```

Comment:
Typo in `example` but I think for all these I would just remove everything after `For example`

Reply:
Removed `For example` from all these comments